src/shared/pipes/asset/asset.pipe.ts

```diff
@@ -50,17 +50,30 @@ export class AssetPipe implements PipeTransform, OnDestroy {
     asset: AssetBankModel | Asset,
     unit: 'trade' | 'base' | 'formatted' = 'formatted'
   ): string | number {
-    const divisor = new Big(10).pow(asset.decimals);
+    const divisor = new Big(10).pow(Number(asset.decimals));
     const tradeUnit = new Big(value).div(divisor);
     const baseUnit = new Big(value).mul(divisor);
 
     switch (unit) {
+      // Whole coin unit without formatting, ex. 0.0023 BTC
       case 'trade': {
         return tradeUnit.toNumber();
       }
+
+      // Base coin unit without formatting, ex. 2000000000 Wei
+      // Type 'string' is returned here to disable scientific notation from JS 'number' Type
       case 'base': {
-        return baseUnit.toNumber();
+        // Set the positive exponent value at and above which toString returns exponential notation.
+        Big.PE = 100;
+
+        let base = baseUnit.toString();
+
+        // Reset
+        Big.PE = 21;
```

Comment:
Shouldn't we read the value, set it and then reset to the value that was read.

Reply:
The default PE is 21, i'm resetting it here because i'm using the same instance of Big.js across the other cases and experienced some weird behaviour if I permanently set it to 100.